### DISCUSSION TITLE
Artifact Manual Removed

### DISCUSSION THREAD

**Discussion by huydinhtran** [accel-sim/accel-sim-framework#173] [Q&A] [answered]
Hi,

I found out that the "release-accelwattch" branch is removed so I can't access the "AccelWattch MICRO'21 Artifact Manual". Could you publish the branch again? Or can I have a copy of the manual?

Thanks,
Huy

**Comment by JRPan** [comment]
It's merged into the release branch. Everything should be available there. If you cannot find anything specific, please let me know.
Thanks.

**Comment by huydinhtran** [comment]
Hi, 

Since the "release-accelwattch" branch was merged with the "release" branch, the instructions in "AccelWattch MICRO'21 Artifact Manual" are outdated. I think some of the scripts (one of them is the setup_environment script) which I used back then required the "release-accelwattch" branch since it was hardcoded. Is it ok if you could bring back that branch?

Thanks.

**Comment by JRPan** [comment, marked as answer]
Vijay still has release-accelwattach here https://github.com/VijayKandiah/accel-sim-framework/tree/release-accelwattch.
